cockpit/tests/api/api-configmap_gravitee_yaml_test.yaml

```diff
@@ -1462,12 +1471,13 @@ tests:
                 responseType: code
                 scopes: openid,profile,email
 
-  - it: should configure onboarding to 5 mins
+  - it: should configure to notifying incomplete onboarding 30 mins later
```

Comment:
As 30 mins is the default value, we may use another value to check the difference.
WDYT ?

Reply:
Good point ! Change to 10 mins